langchain/src/tools/duckduckgo.ts

```diff
@@ -0,0 +1,26 @@
+import { search as DDGsearch, SafeSearchType } from 'duck-duck-scrape';
```

Comment:
Use dynamic import whenever possible, as this is being imported into the barrel file